File: src/kyu_8/simple_fun_number_1_seats_in_theater/python/simple_fun_number_1_seats_in_theater_test.py
@test.describe("Basic tests")
def fxd():
    @test.it('')
    def check():
        test.assert_equals(seats_in_theater(16,11,5,3) , 96)
        test.assert_equals(seats_in_theater(1,1,1,1) , 0)
        test.assert_equals(seats_in_theater(13,6,8,3) , 18)
        test.assert_equals(seats_in_theater(60,100,60,1) , 99)
        test.assert_equals(seats_in_theater(1000,1000,1000,1000) , 0)

File: src/kyu_8/simple_fun_number_1_seats_in_theater/csharp/simple_fun_number_1_seats_in_theater_test.cs
namespace Solution
{
  using NUnit.Framework;

  using System;

  [TestFixture]
  public class Tests
  {
    [Test]
    public void ExampleTests()
    {
      Assert.AreEqual(96, Kata.SeatsInTheater(16, 11, 5, 3));

      Assert.AreEqual(0, Kata.SeatsInTheater(1, 1, 1, 1));

      Assert.AreEqual(18, Kata.SeatsInTheater(13, 6, 8, 3));

      Assert.AreEqual(99, Kata.SeatsInTheater(60,100,60,1));

      Assert.AreEqual(0, Kata.SeatsInTheater(1000,1000,1000,1000));
    }
  }
}